nuspec/chocolatey/ChocolateyGUI.nuspec

```diff
@@ -52,7 +52,7 @@ All release notes for Chocolatey GUI can be found on the GitHub site - https://g
     <file src="chocolateyInstall.ps1" target="tools"/>
     <file src="chocolateyUninstall.ps1" target="tools"/>
     <file src="..\..\BuildArtifacts\ChocolateyGUI.msi" target="tools"/>
-    <file src="..\..\LICENSE" target="tools\LICENSE"/>
+    <file src="..\..\LICENSE" target="tools\LICENSE.txt"/>
```

Comment:
This change won't result in what you're trying to achieve.

When the last segment don't match the actual name of the imported file, then a directory will be created.

![image](https://user-images.githubusercontent.com/1474648/78472903-bfedd780-773c-11ea-85c3-b78105d5925a.png)

Reply:
TIL!

@AdmiringWorm @gep13 do we want to rename the LICENSE file inside the repo to LICENSE.txt or do we want to copy the file over into the tools folder with the correct extension?

Reply:
I have no objections to renaming the actual file to include the extension.

Reply:
I’ll try and pick that up later today or tonight.

Reply:
The file have been renamed. All other locations pointing to the license file have been updated as well.